makefile.cargo

```diff
@@ -8,28 +8,27 @@ CONFIGURE_FLAGS := \
 
 ifeq (eabi,$(findstring eabi,$(TARGET)))
```

Comment:
I suppose so.  Any idea what they didn't check do a HOST and TARGET compare to begin with?  Something specific to ARM?